doctr/transforms/modules/pytorch.py

```diff
@@ -11,16 +11,16 @@
 from torchvision.transforms import functional as F
 from torchvision.transforms import transforms as T
 
-__all__ = ['Resize']
+__all__ = ['Resize', 'RandomGaussianNoise']
 
 
 class Resize(T.Resize):
     def __init__(
-        self,
-        size: Tuple[int, int],
-        interpolation=F.InterpolationMode.BILINEAR,
-        preserve_aspect_ratio: bool = False,
-        symmetric_pad: bool = False,
+            self,
+            size: Tuple[int, int],
+            interpolation=F.InterpolationMode.BILINEAR,
+            preserve_aspect_ratio: bool = False,
+            symmetric_pad: bool = False,
```

Comment:
please don't change things that aren't related to this PR :pray: 
(also this makes the indentation wrong)

Reply:
Yeah, actually I had added a class for adding Gaussian noise in the same script. Due to automatic collapsing of the import I couldnt figure out that the refactoring had changed the spacing :)  Will take care of it for the next time.